dev/WindowsAppRuntime_BootstrapDLL/WindowsAppRuntime-Licensing.cpp

```diff
@@ -6,9 +6,125 @@
 
 #include <msixlicensing.h>
 
-STDAPI WindowsAppRuntime_InstallLicenses() noexcept try
+static std::filesystem::path GetPathForPackage(
+    PCWSTR packageFullName)
 {
-    RETURN_IF_FAILED(MsixInstallLicenses());
+    // Get the package's path
+    //
+    // NOTE: Use GetStagedPackagePathByFullName as don't care if the package is registered.
+    //
+    // NOTE: Don't use GetStagedPackagePathByFullName2 as we want the *install* location
+    //       (not e.g. modifiable location). Plus Get...2() is new in 20H1 (too new for us)
+    //       where Get...() exists down to Win81. In this case simplest is bestest.
+    wil::unique_cotaskmem_string packagePathBufferDynamic;
+    uint32_t packagePathLength{};
+    const auto rc{ GetStagedPackagePathByFullName(packageFullName, &packagePathLength, nullptr) };
+    if (rc != ERROR_INSUFFICIENT_BUFFER)
+    {
+        THROW_HR_MSG(HRESULT_FROM_WIN32(rc), "PackageFullName:%ls", packageFullName);
+    }
+    auto packagePath{ wil::make_cotaskmem_string(nullptr, packagePathLength) };
+    THROW_IF_WIN32_ERROR(GetStagedPackagePathByFullName(packageFullName, &packagePathLength, packagePath.get()));
+    std::filesystem::path path{ packagePath.get() };
+    return path;
+}
+
+static HMODULE LoadMsixLicensingApisAtPath(
```

Comment:
This method should return `wil::unique_hmodule` so the callers don't have to wonder about lifetime.

Reply:
I thought about that but thought there was general preference to return e.g. HMODULE and let callers decide how to handle it. Or is that more relevant for other codebases and here we can/should e.g. go pervasively wil::unique_hmodule and not HMODULE?